docroot/sites/tippie.uiowa.edu/modules/tippie_migrate/config/split/migrate_plus.migration.tippie_articles_redirects.yml

```diff
@@ -0,0 +1,37 @@
+langcode: en
+status: true
+dependencies: {  }
+id: tippie_articles_redirects
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags:
+  - tippie
+migration_group: sitenow_migrate
+label: 'Article Redirects'
+source:
+  plugin: tippie_articles
+  node_type: news
+  constants:
+    redirect_prefix: 'internal:/node/'
+    dash: '-'
```

Comment:
Is this getting used anywhere? I'm assuming it's an artifact from earlier migrations--I have a vague memory of needing it at some point for a plugin at some point.

Reply:
copy pasta from iowajpec i think

Reply:
I think we've been copy-pastaing it since the thesis defense migration, and I just now noticed it. Removed it as a bit of cleanup prior to installing/testing locally, and everything still looks good.